project/frontend/src/contexts/FirestoreContext.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+import * as firebase from 'firebase/app';
+import 'firebase/firestore';
+
+function _initFirestore() {
+  const firebaseConfig = {
+    apiKey: 'AIzaSyC0Q4CyO-BM4M5jPvL3ayJ09RfymZYQjhs',
+    authDomain: 'mak-step-2020.firebaseapp.com',
+    databaseURL: 'https://mak-step-2020.firebaseio.com',
+    projectId: 'mak-step-2020',
+    storageBucket: 'mak-step-2020.appspot.com',
+    messagingSenderId: '21423804760',
+    appId: '1:21423804760:web:4fd3b3fdef792c51b3552d',
```

Comment:
These should be in a gitignore

Reply:
I'm not sure if it is necessary to obscure the whole object. From the firebase docs: `The Firebase config object contains unique, but *non-secret* identifiers for your Firebase project.` Would it be sufficient to obscure only the api key using the .env file we've previously created?

Reply:
I would create a firebaseAuth file of some sort to gitignore and put all these values in there. Then we can import those here and this object would be 
`const config = {
  apiKey: API_KEY,
  authDomain: AUTH_DOMAIN,
  databaseURL: DATABASE_URL,
  projectId: PROJECT_ID,
  storageBucket: STORAGE_BUCKET,
  messagingSenderId: MESSAGING_SENDER_ID,
  appId: APP_ID
};`